addons/event/models/event_event.py

```diff
@@ -397,11 +397,13 @@ def _compute_event_mail_ids(self):
                 continue
 
             # lines to keep: those with already sent emails or registrations
-            mails_toremove = event._origin.event_mail_ids.filtered(lambda mail: not mail.mail_sent and not(mail.mail_registration_ids))
-            command = [(3, mail.id) for mail in mails_toremove]
+            mails_toremove = event.event_mail_ids.filtered(
+                lambda mail: not(mail._origin.mail_sent or mail._origin.mail_registration_ids)
+            )
+            command = [Command.unlink(mail.id) for mail in mails_toremove]
             if event.event_type_id.use_mail_schedule:
                 command += [
-                    (0, 0, {
+                    Command.create({
```

Comment:
what's the difference ?
same for (3, ...) and unlink ?
Is that all the point of that record were not really removed ?

Reply:
No the trick is with "_origin". Command.create is just a new feature and it was recommended to update new code with this syntax

Reply:
"it was recommended to update new code with this syntax" Not by SM standards :pirate_flag: :skull_and_crossbones: 

Anyway you can use those Commands if you hate your team. It makes us very very sad.

PS: this is a joke, better use Commands indeed :)